ugvc/mrd/srsnv_plotting_utils.py

```diff
@@ -889,19 +888,19 @@ def plot_qual_per_feature(
         for j, label in enumerate(labels_dict):
             if df[feature].dtype == bool:
                 if j == 0:
-                    plt.figure(figsize=(6, 3))
+                    plt.figure(figsize=(8, 6))
                 _ = (
                     df[df["label"] == label][feature]
                     .astype(int)
                     .hist(bins=20, alpha=0.5, label=labels_dict[label], density=True)
                 )
             elif df[feature].dtype in {"category", "object"}:
                 if j == 0:
-                    plt.figure(figsize=(16, 4))
+                    plt.figure(figsize=(8, 6))
                 _ = df[df["label"] == label][feature].hist(bins=20, alpha=0.5, label=labels_dict[label], density=True)
```

Comment:
While you're at it - 20 bins is often ill-fitted to the data. For categories the bins need to be the number of unique categories, for integer values they should often be integers. 

Consider tweaking this, or even harcoding bins for the various features and only using a default of 20 for features that were not encoded.

If it would cause a significant delay then it can wait for another opportuinity.

Reply:
done